src/toil/leader.py

```diff
@@ -1107,10 +1107,11 @@ def innerLoop(jobStore, config, batchSystem, toilState, jobBatcher, serviceManag
                     # Remove the job
                     if jobGraph.remainingRetryCount > 0:
                         jobBatcher.issueJob(JobNode.fromJobGraph(jobGraph))
-                        logger.debug("Job: %s is empty, we are scheduling to clean it up", jobGraph.jobStoreID)
+                        logger.debug("Job: %s is empty, we are scheduling to clean it up", jobGraph)
                     else:
                         jobBatcher.processTotallyFailedJob(jobGraph)
-                        logger.warn("Job: %s is empty but completely failed - something is very wrong", jobGraph.jobStoreID)
+                        logger.critical("Job: %s is empty but completely failed - something is "
```

Comment:
Then again, if it did happen would we want to continue the pipeline anyway? If this is a realistic edge case then we probably want to continue the rest of the jobs. That failure would happen again on restart so throwing an error whenever it occurred would prevent a user from making any progress & make them start a completely new run.